rocAL/rocAL_pybind/amd/rocal/pipeline.py

```diff
@@ -181,7 +183,7 @@ def copyImage(self, array):
     def copyToTensor(self, array,  multiplier, offset, reverse_channels, tensor_format, tensor_dtype):
 
         b.rocalCopyToOutputTensor(self._handle, ctypes.c_void_p(array.data_ptr()), tensor_format, tensor_dtype,
-                                    multiplier[0], multiplier[1], multiplier[2], offset[0], offset[1], offset[2], (1 if reverse_channels else 0))
+                                    multiplier[0], multiplier[1], multiplier[2], offset[0], offset[1], offset[2], (1 if reverse_channels else 0), (1 if self._normalization_on_device else 0))
```

Comment:
I think we need to discuss how to support this for CPU pipeline. 
we need to allocate pinned memory if the output requested is on device. That way, we can eliminate extra host to device copy.

Reply:
Changed the API calls according to the changes discussed in the call